javascript/i18n/phonenumbers/demo.js

```diff
@@ -46,75 +46,82 @@ function phoneNumberParser() {
     output.append('****Parsing Result:****\n');
     output.append(goog.json.serialize(new goog.proto2.ObjectSerializer(
         goog.proto2.ObjectSerializer.KeyOption.NAME).serialize(number)));
-    output.append('\n\n****Validation Results:****');
+    output.append('\n\n****Possibility Results:****');
```

Comment:
Please do not rename this header. "Possibility results" doesn't make sense, and checking if a number is possible is a type of validation.

Reply:
True. Gone with motion of product terminology.
Clubbed the results.